jumpstart-launch-tasks.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+read -p "What is the site shortname on ACSF (e.g., \"jumpstartlockss\", no quotes)" sitename
+# Baby, what time is it?
+timestamp=$(date +%Y%m%d%H%M%S)
+howardpass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
+lindseypass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
+adminpass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
+drush @acsf.cardinald7.$sitename -y en googleanalytics site_verify xmlsitemap xmlsitemap_menu xmlsitemap_engines
+drush @acsf.cardinald7.$sitename -y vset --always-set block_cache 1
+drush @acsf.cardinald7.$sitename -y vset --always-set cache 1
+drush @acsf.cardinald7.$sitename -y vset --always-set cache_lifetime 0
+drush @acsf.cardinald7.$sitename -y vset --always-set features_rebuild_on_flush FALSE
+drush @acsf.cardinald7.$sitename -y vset --always-set page_cache_maximum_age 21600
+drush @acsf.cardinald7.$sitename -y vset --always-set preprocess_css 1
+drush @acsf.cardinald7.$sitename -y vset --always-set preprocess_js 1
+drush @acsf.cardinald7.$sitename -y vset --always-set page_compression 1
```

Comment:
Per Acquia Support, this should be disabled. They say it interferes with Varnish.